src/main/java/de/tum/in/www1/artemis/domain/iris/settings/IrisSubSettings.java

```diff
@@ -40,6 +40,7 @@
 // @formatter:off
 @JsonSubTypes({
     @JsonSubTypes.Type(value = IrisChatSubSettings.class, name = "chat"),
+    @JsonSubTypes.Type(value = IrisLectureIngestionSubSettings.class, name = "ingestion"),
```

Comment:
I'd suggest to name this "lecture_ingestion", as we will likely have more than one type of ingestion in the future.